_posts/2020-04-21-0.17.0-release.md

```diff
@@ -0,0 +1,197 @@
+from [**79 distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 0.16.0 release, two committers have joined the Project Management
+Committee (PMC):
+
+* [Neal Richardson][4]
+* [François Saint-Jacques][5]
+
+Thank you for all your contributions!
+
+## Columnar Format Notes
+
+A [C-level Data Interface][6] was designed to ease data sharing inside a single process. It allows different runtimes or libraries to share Arrow data using a well-known binary layout and metadata representation, without any copies. The C++ library now includes an implementation of this specification, and Python and R have bindings to that implementation.
+
+## Arrow Flight RPC notes
+
+* Adopted new DoExchange bi-directional data RPC
+* ListFlights supports being passed a Criteria argument in Java/C++/Python. This allows applications to search for flights satisfying a given query.
+* Custom metadata can be attached to errors that the server sends to the client, which can be used to encode richer application-specific information.
+* A number of minor bugs were fixed, including proper handling of empty null arrays in Java and round-tripping of certain Arrow status codes in C++/Python.
+
+## C++ notes
+
+### Feather V2
+
+The "Feather V2" format based on the Arrow IPC file format was developed.
+Feather v2 features full support for all Arrow data types,
+and resolves the 2GB per-column limitation for large amounts of string data that
+the [original Feather implementation][13] had.
+Feather V2 also introduces experimental IPC message compression using LZ4 frame format or ZSTD. This will be formalized later in the Arrow format.
+
+### C++ Datasets
+
+TODO: SUMMARIZE C++ DATASETS WORK
+
+### C++ Parquet notes
+
+* Complete support for writing nested types to Parquet format was completed. The legacy code can be accessed through parquet write option C++ and an environment variable in Python. Read support will come in a future release.
+* The BYTE_STREAM_SPLIT encoding was implemented for floating-point types. It helps improve the efficiency of memory compression for high-entropy data.
+* Expose Parquet schema field_id as Arrow field metadata
+* Support for DataPageV2
+
+### C++ build notes
+
+* We continued to make the core C++ library build simpler and faster. Among the improvements are the removal of the dependency on Thrift Compiler at build time; while Parquet still requires `thrift-cpp`, its dependencies are much lighter. We also further reduced the number of build configurations that require `boost`, and when `boost` is needed to be built, we only download the components we need, reducing the size of the `boost` bundle by 90%.
+* TODO: ARM64 platform notes?
+* TODO: LLVM 8 upgrade?
```

Comment:
@kou?

Reply:
Will do in a few days.
(If someone can take over this, please do.)